portfolio/src/main/webapp/bandGen.js

```diff
@@ -5,20 +5,28 @@ function getMood() {
 }
 
 function generateBand() {
-    const NUMBER_OF_BANDS = 2;
+    const NUMBER_OF_BANDS = 5;
```

Comment:
nit: From the [style guide](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names): "Constants in function scopes are still named in `lowerCamelCase`"